src/app/tests/TestReadInteraction.cpp

```diff
@@ -798,18 +801,16 @@ void TestReadInteraction::TestReadHandlerSetMaxReportingInterval()
     auto * engine = chip::app::InteractionModelEngine::GetInstance();
     EXPECT_EQ(engine->Init(&GetExchangeManager(), &GetFabricTable(), gReportScheduler), CHIP_NO_ERROR);
 
-    uint16_t kIntervalInfMinInterval = 119;
-    uint16_t kMinInterval            = 120;
-    uint16_t kMaxIntervalCeiling     = 500;
+    uint16_t kMinInterval        = 120;
+    uint16_t kMaxIntervalCeiling = 500;
```

Comment:
can we add units to these constant names?
Should these be constexpr given `k` naming?

Reply:
they are not const, these are previous code, I have updated with test prefix and get rid of k, thanks